src/docs/sdk/event-payloads/transaction.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Transaction Payloads
+---
+
+Transactions are used to send tracing events to Sentry.
+
+Transactions must be wrapped in an [Envelope](/sdk/envelopes/) and therefore also be sent to the Envelope endpoint.
+
+<Alert title="Note on backwards compatibility" level="info">
+
+Envelopes are a new format and therefore only work on Sentry >= 10. More information can be found on the [Envelope](/sdk/envelopes/) page.
+</Alert>
+
+## Anatomy
+
+A Transaction is basically a [Span](/sdk/event-payloads/span) combined with an [Event](/sdk/event-payloads/). When using 
+tracing with our SDKs you usually create a Span tree, the root node and therefore the whole tree is considered to be the Transaction.
+So technically a Transaction is just a Span. A Transaction must also have a `contexts.trace` (which contains some data of the [Span](/sdk/event-payloads/span)) and some other
+properties that will be covered in the next section.
+
+Transactions is an [Events](/sdk/event-payloads/) which will be enriched with data from the [Span](/sdk/event-payloads/span).
+We are only going to list here what is important for a Transaction.
+
+import "./properties/type.mdx"
+
+import "./properties/span_start_timestamp.mdx"
+
+import "./properties/span_timestamp.mdx"
+
+### contexts.trace
+
+A Transaction has to have a specific `contexts.trace` entry that contains data from the [Span](/sdk/event-payloads/span).
```

Comment:
I believe this should be documented similar to other `contexts`, specially because we're sending `contexts.trace` also in error events / `captureMessage`.

(I've asked about this on Slack earlier this week after seeing `contexts.trace` in the payload from `captureMessage`)